sonar-project.properties

```diff
@@ -1,5 +1,5 @@
-sonar.projectKey=open-cluster-management_cm-cli
-sonar.projectName=cm-cli
+sonar.projectKey=open-cluster-management_clusteradm
```

Comment:
let's remove sonar file until we decide whether we also need to integrate sonar here.

Reply:
Done